<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>우리들의 국민상회 오픈챗방 카톡 메시지 보내는 프로그램</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="msg0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSaveMsg0">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMsg0">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="msg1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnSaveMsg1">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMsg1">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="msg2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btnSaveMsg2">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMsg2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="msg3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnSaveMsg3">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMsg3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>241</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="rpt_msg0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="btnSaveRptMsg0">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendRptMsg0">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="rptTime0">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="currentText">
        <string>1시간 마다 반복</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>반복시간선택</string>
       </property>
       <item>
        <property name="text">
         <string>1시간 마다 반복</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2시간 마다 반복</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3시간 마다 반복</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4시간 마다 반복</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnRptSendMsg0">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>반복카톡보내기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRptStopMsg0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>반복카톡중지</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="msg4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="btnSaveMsg4">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMsg4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="msg5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QPushButton" name="btnSaveMsg5">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMsg5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>290</y>
     <width>241</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="rpt_msg1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QPushButton" name="btnSaveRptMsg1">
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>메시지저장</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendRptMsg1">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>카톡보내기</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="rptTime1">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="currentText">
        <string>1시간 마다 반복</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="placeholderText">
        <string>반복시간선택</string>
       </property>
       <item>
        <property name="text">
         <string>1시간 마다 반복</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2시간 마다 반복</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3시간 마다 반복</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4시간 마다 반복</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QPushButton" name="btnRptSendMsg1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>반복카톡보내기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRptStopMsg1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>반복카톡중지</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>msg0</tabstop>
  <tabstop>btnSaveMsg0</tabstop>
  <tabstop>btnSendMsg0</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
